fili-core/src/main/resources/moduleConfig.properties

```diff
@@ -86,8 +86,8 @@ bard__max_results_without_filters = 10000
 bard__default_per_page = 10000
 
 # URLs for druid nodes
-bard__druid_broker = [SET ME IN APPLICATION CONFIG]
-bard__druid_coord = [SET ME IN APPLICATION CONFIG]
+# bard__druid_broker = [SET ME IN APPLICATION CONFIG]
+# bard__druid_coord = [SET ME IN APPLICATION CONFIG]
```

Comment:
I don't think we should be commenting these out. The entire purpose of having them be uncommented is that if the customer forgets to set this, the error message will tell them that the setting for this is invalid, and when the error message prints the provided setting, they'll get a hint as to where to put the proper setting.

Reply:
Then why do we even bother to be backward compatible then? This default will always be chosen over the deprecated ui and non ui settings.

Reply:
@archolewa  To clarify, these being set was breaking this because we were using their UNSETness to indicate backwards compatibility.  Instead they were showing as set, and blowing up unmigrated isntances.

Reply:
@michael-mclawhorn Ah ok that makes sense, I forgot about that.